src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,222 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CalculatorTest {
+    private static Calculator calculator;
+    private static final double VALUE1 = 34.324;
+    private static final double VALUE2 = -23.21;
+    private static final double DELTA = 0.0001;
+    private static final double ZERO = 0;
+    private static final char PLUS = '+';
+    private static final char SUBTRACTION = '-';
+    private static final char MULTIPLY = '*';
+    private static final char DIVIDE = '/';
+    private static final char POW = '^';
+    private static final char INVALID_OPERATOR = '#';
+    private static double expected;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void calculateAdditionWithTwoPositiveOperands_Ok() {
+        expected = 68.648;
+        assertEquals(expected, calculator.calculate(VALUE1,VALUE1,PLUS), DELTA);
```

Comment:
якщо вже створювати змінну expected, то я б ще створив поле actual і використовував його в тестових методах assert, так наче краще б читалось
```suggestion
        expected = 68.648;
        actual = calculator.calculate(VALUE1,VALUE1,PLUS);
        assertEquals(expected, actual, DELTA);
```

Reply:
учел